src/lib/components/drs/DrsMain.js

```diff
@@ -0,0 +1,312 @@
+import '@fontsource/roboto';
+import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+import {
+  Switch,
+  Route,
+  useHistory
+} from "react-router-dom";
+import { Snackbar } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { format } from 'date-fns';
+import DrsIndex from './pages/DrsIndex';
+import DrsObjectForm from './pages/DrsObjectForm';
+import DrsApiCaller from './utils/DrsApiCaller';
+import _ from 'lodash';
+import FormViewType from '../../model/common/FormViewType';
+
+const DrsMain = props => {
+
+  const emptyDrsObject = () => {
+    let date = new Date();
+    date.setSeconds(0, 0);
+    let emptyDrsObject = {
+      id: '',
+      description: '',
+      created_time: format(date, "yyyy-MM-dd'T'HH:mm:ss'Z'"),
+      mime_type: '',
+      name: '',
+      size: '',
+      updated_time: format(date, "yyyy-MM-dd'T'HH:mm:ss'Z'"),
+      version: '',
+      is_bundle: false,
+      aliases: [],
+      checksums: [],
+      drs_object_children: [],
+      drs_object_parents: [],
+      file_access_objects: [],
+      aws_s3_access_objects: []
+    }
+    return emptyDrsObject;
+  }
+
+  const emptyDisplayChecksumTypes = () => {
+    return {
+      md5: true,
+      sha1: true,
+      sha256: true
+    }
+  }
+
+  /* ##################################################
+   * STATE VARIABLES
+   * ################################################## */
+
+  const [drsObjectsList, setDrsObjectsList] = useState([]);
+  const [activeDrsObject, setActiveDrsObject] = useState(emptyDrsObject());
```

Comment:
If the user refreshes the browser while on a SHOW page, the page will not be repopulated with the data corresponding to the DrsObject ID in the path. Instead, the page will display the SHOW page of an empty DRS Object.